wiki/Tournaments/Official_support/en.md

```diff
@@ -0,0 +1,166 @@
+
+Individual players who are unhappy with their screening outcome should be told to consult [accounts@ppy.sh](mailto:accounts@ppy.sh) via email.
+
+**Allowing users that have failed screening to play in your tournament will result in an immediate withdrawal of support and will result in future requests for support to be denied.**
+
+## Designing your profile badge
+
+Profile badge images must adhere to the following standards:
+
+- The badge image file MUST be a PNG.
+- The badge image MUST be **172x80**px in dimensions.
+- The badge MUST clearly display the logo, motif or name of your tournament.
+- The badge MUST be cleanly designed and of reasonable visual quality.
+
+Please consult the following examples of approved badges:
+
+![](https://assets.ppy.sh/profile-badges/flc2-2020@2x.png)
+![](https://assets.ppy.sh/profile-badges/irt2-2020@2x.png)
+
+Badge images that do not meet these standards will be refused with given reasoning, and you may be asked to completely redesign your badge in some cases. This is fairly rare, but should be accounted for.
+
+## Requesting support
+
+To start with, the request for support **must be made well before the tournament actually commences play.** 
+
+There are no specific timelines for this, but we recommend at least two to three weeks before play commences to account for delays in the screening process. 
+
+You may be required to postpone the start of your tournament during periods of high delay, but the osu!team will make efforts to ensure this does not happen.
+
+Please read the criteria laid out below, then send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:
+
+- The name of your tournament
+- A brief description of your tournament, including its scoring type and any other appropriate features
+- Any applicable dates for your tournament
+- Any links to public off-site chatrooms or servers used for the event (Discord, etc.)
+- A link to the forum thread located under the [Tournaments subforum](https://osu.ppy.sh/community/forums/55) that announces/details your tournament
+- Any links to previous iterations of the same tournament (only where applicable)
+- A list of the players involved in running the tournament
+- A list of players who have registered for the tournament
+
+Please allow **at least a week** for a response. We will try our best to handle all requests within a week, but we occasionally have large spikes in workload that can make this impossible.
+
+### Criteria
+
+Community-run tournaments which abide by the following hard criteria are eligible for support:
+
+- The tournament must have at least 64 teams/players minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
```

Comment:
I understand one new requirement which is a minimum of 64 players, however 64 teams? If I were to host a single-elimination tournament that has over 200 players signing up but only acquiring 40 teams, wouldn't that make the tournament ineligible for official support? I believe the requirement of 64 teams either isn't clear enough or severely punishes single-elimination tournaments.

Reply:
Should be an oversight, will be corrected later.

Reply:
Also in case you missed it, even without fixing that oversight, your tournament can still have 16 players or teams, as long as it's double-elimination.

Reply:
You can still have less than 64 teams and use a Ro16 double elimination. The tournament would still qualify in that case. I don't think this was an oversight.

Reply:
The thing is the **OR** there, because if the tournament isnt double elimination (like single elimination or swiss-system for example) it is forced to have 64 teams.

Reply:
And that's perfectly fine! Having 64 players in teams of lets say 8 each means you have 8 teams. Single elimination of 8 teams should not be badged imo.

Reply:
I think it goes more on the wording and the concern of @megu-osu points, if we go by number of players (that is totally fine) how this converts into teams? 16 teams of 4 seems like a bare minimum, but for example if we have something similar to test tournament, we would have 4 teams of 16.

The solution should be having this clear on the "teams" side, something like:
```suggestion
- The tournament must have at least 64 players/32 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
```

Also reminding that this would affect other types of tournament systems, like swiss.